src/Data.cpp

```diff
@@ -100,21 +92,14 @@ Data::Data(Configuration cfg)
   , m_recordingRecordedDuration{0}
   , m_epgMinTime{time(nullptr)}
   , m_epgMaxTime{time(nullptr) + 3600}
-  , m_epgMaxDays{cfg.epgMaxDays}
+  , m_epgMaxDays{EpgMaxDays()}
   , m_bEGPLoaded{false}
   , m_iLastStart{0}
   , m_iLastEnd{0}
-  , m_streamQuality{static_cast<ApiManager::StreamQuality_t>(cfg.streamQuality)}
-  , m_fullChannelEpgRefresh{cfg.fullChannelEpgRefresh}
-  , m_loadingsRefresh{cfg.loadingsRefresh}
-  , m_keepAliveDelay{cfg.keepAliveDelay}
-  , m_epgCheckDelay{cfg.epgCheckDelay}
-  , m_useH265{cfg.useH265}
-  , m_useAdaptive{cfg.useAdaptive}
-  , m_showLockedChannels{cfg.showLockedChannels}
-  , m_showLockedOnlyPin{cfg.showLockedOnlyPin}
-  , m_currentStreamIsLive{false}
-  , m_manager{std::move(cfg.userName), std::move(cfg.password), std::move(cfg.deviceId), std::move(cfg.productId)}
+  , m_manager{std::move(kodi::GetSettingString("userName")),
```

Comment:
I'm not certain, but doesn't `std::move` prohibit copy elision here?

Reply:
Also not sure, thing to remove it, normally return settings a created "std::string" like with "cfg" before, but with "const std::string &" as value on class makes itnot direct sense.